cam0:
  T_cam_imu:
  - [0.999975, 0.007038, -0.000812, -0.037983]
  - [-0.007035, 0.999969, 0.003395, 0.002513]
  - [0.000836, -0.003389, 0.999994, -0.001260]
  - [0.000000, 0.000000, 0.000000, 1.000000]
  cam_overlap: [1]
  camera_model: pinhole
  distortion_coeffs: [0.231222, -0.784899, -0.003257, -0.000105, 0.917205]
  distortion_model: radtan
  intrinsics: [520.908620, 521.007327, 325.141442, 249.701764]
  resolution: [640, 480]
  rostopic: /camera/rgb/image_color
rgb:
  T_rgb_cam0:
  - [0.999968, 0.003327, -0.007251, -0.001814]
  - [-0.003260, 0.999953, 0.009178, -0.028674]
  - [0.007281, -0.009154, 0.999932, -0.020372]
  - [0.000000, 0.000000, 0.000000, 1.000000]
  distortion_coeffs: [0.231222, -0.784899, -0.003257, -0.000105, 0.917205]
  distortion_model: radtan
  intrinsics: [520.908620, 521.007327, 325.141442, 249.701764]
  resolution: [640, 480]
depth:
  distortion_coeffs: [0.231222, -0.784899, -0.003257, -0.000105, 0.917205]
  distortion_model: radtan
  depth_correction: [0.0, 0.0]
  intrinsics: [520.908620, 521.007327, 325.141442, 249.701764]
  resolution: [640, 480]
vicon:
  T_vicon_cam:
  - [0.999885, 0.014491, -0.004467, 0.053458]
  - [-0.014337, 0.999362, 0.032704, -0.006089]
  - [0.004938, -0.032636, 0.999455, -0.003491]
  - [0.000000, 0.000000, 0.000000, 1.000000]
